src/Codeception/Lib/Connector/Laravel5.php

```diff
@@ -223,6 +222,28 @@ private function initialize($request = null)
         $this->module->setApplication($this->app);
     }
 
+    /**
+     * Get the Laravel app url.
+     * @return string
+     * @throws ModuleConfig
+     */
+    private function getUrl()
+    {
+        return (isset($this->module->config['url'])) ? $this->module->config['url'] :  $this->getUrlFromApp();
```

Comment:
please add `url` to list of config options inside `Laravel5` helper with a default value.
All config options should have default values, so `isset` is not needed